addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -618,6 +618,17 @@ def is_valid_location_dest_id(self, location_ref):
         return all([pick.is_valid_location_dest_id(location=location)
                     for pick in all_done_pickings])
 
+    def _check_unpickable_item(self):
+        """ Checks if the picking type of the batch is allowed to handle
+            unpickable items.
+            If it is not, it raises an error asking to contact team leader.
+        """
+        if not all(self.picking_type_ids.mapped('u_handle_unpickable_items')):
+            raise ValidationError(
+                _('Contact your team leader.')
```

Comment:
Exception messages should describe errors and failures. Then it should be up to the caller to catch them and decide what to do. The above message is just weird.

Reply:
it's part of the AC's that they want this message by the looks of things =s

Reply:
I agree the message is a bit weird, I have changed it to something more meaningful and waiting for PO confirmation.